```
ghidra-aarch64-darwin> Successfully started process 'Gradle build daemon'
ghidra-aarch64-darwin> An attempt to start the daemon took 0.503 secs.
ghidra-aarch64-darwin> 
ghidra-aarch64-darwin> FAILURE: Build failed with an exception.
ghidra-aarch64-darwin> 
ghidra-aarch64-darwin> * What went wrong:
ghidra-aarch64-darwin> Unable to start the daemon process.
ghidra-aarch64-darwin> This problem might be caused by incorrect configuration of the daemon.
ghidra-aarch64-darwin> For example, an unrecognized jvm option is used.
ghidra-aarch64-darwin> Please refer to the User Manual chapter on the daemon at https://docs.gradle.org/7.6.1/userguide/gradle_daemon.html
ghidra-aarch64-darwin> Process command line: /nix/store/vznwx8md3ma3nfviy23hl7905n1hmn6k-zulu17.34.19-ca-jdk-17.0.3/bin/java --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.lang.invoke=ALL-UNNAMED --add-opens=java.prefs/java.util.prefs=ALL-UNNAMED --add-opens=java.base/java.nio.charset=ALL-UNNAMED --add-opens=java.base/java.net=ALL-UNNAMED --add-opens=java.base/java.util.concurrent.atomic=ALL-UNNAMED -Xmx2G -Dfile.encoding=UTF-8 -Duser.country=US -Duser.language=en -Duser.variant -cp /nix/store/n4mjqb2pa0bkvahaww540jln9rdkrk5f-gradle-7.6.1/lib/gradle/lib/gradle-launcher-7.6.1.jar org.gradle.launcher.daemon.bootstrap.GradleDaemon 7.6.1
ghidra-aarch64-darwin> Please read the following process output to find out more:
ghidra-aarch64-darwin> -----------------------
ghidra-aarch64-darwin> Picked up JAVA_TOOL_OPTIONS: -Duser.home='/private/tmp/nix-build-ghidra-10.3.drv-0/home'
ghidra-aarch64-darwin> 
ghidra-aarch64-darwin> FAILURE: Build failed with an exception.
ghidra-aarch64-darwin> 
ghidra-aarch64-darwin> * What went wrong:
ghidra-aarch64-darwin> java.net.SocketException: Operation not permitted
ghidra-aarch64-darwin> 
ghidra-aarch64-darwin> * Try:
ghidra-aarch64-darwin> > Run with --info or --debug option to get more log output.
ghidra-aarch64-darwin> > Run with --scan to get full insights.
ghidra-aarch64-darwin> 
ghidra-aarch64-darwin> * Exception is:
ghidra-aarch64-darwin> org.gradle.api.UncheckedIOException: java.net.SocketException: Operation not permitted
ghidra-aarch64-darwin>         at org.gradle.internal.UncheckedException.throwAsUncheckedException(UncheckedException.java:62)
ghidra-aarch64-darwin>         at org.gradle.internal.UncheckedException.throwAsUncheckedException(UncheckedException.java:41)
ghidra-aarch64-darwin>         at org.gradle.internal.remote.internal.inet.TcpIncomingConnector.accept(TcpIncomingConnector.java:62)
ghidra-aarch64-darwin>         at org.gradle.launcher.daemon.server.DaemonTcpServerConnector.start(DaemonTcpServerConnector.java:86)
ghidra-aarch64-darwin>         at org.gradle.launcher.daemon.server.Daemon.start(Daemon.java:169)
ghidra-aarch64-darwin>         at org.gradle.launcher.daemon.bootstrap.DaemonMain.doAction(DaemonMain.java:125)
ghidra-aarch64-darwin>         at org.gradle.launcher.bootstrap.EntryPoint.run(EntryPoint.java:50)
ghidra-aarch64-darwin>         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
ghidra-aarch64-darwin>         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
ghidra-aarch64-darwin>         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
ghidra-aarch64-darwin>         at java.base/java.lang.reflect.Method.invoke(Method.java:568)
ghidra-aarch64-darwin>         at org.gradle.launcher.bootstrap.ProcessBootstrap.runNoExit(ProcessBootstrap.java:60)
ghidra-aarch64-darwin>         at org.gradle.launcher.bootstrap.ProcessBootstrap.run(ProcessBootstrap.java:37)
ghidra-aarch64-darwin>         at org.gradle.launcher.daemon.bootstrap.GradleDaemon.main(GradleDaemon.java:22)
ghidra-aarch64-darwin> Caused by: java.net.SocketException: Operation not permitted
ghidra-aarch64-darwin>         at java.base/sun.nio.ch.Net.bind0(Native Method)
ghidra-aarch64-darwin>         at java.base/sun.nio.ch.Net.bind(Net.java:555)
ghidra-aarch64-darwin>         at java.base/sun.nio.ch.ServerSocketChannelImpl.netBind(ServerSocketChannelImpl.java:337)
ghidra-aarch64-darwin>         at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:294)
ghidra-aarch64-darwin>         at java.base/sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:89)
ghidra-aarch64-darwin>         at java.base/sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:81)
ghidra-aarch64-darwin>         at org.gradle.internal.remote.internal.inet.TcpIncomingConnector.accept(TcpIncomingConnector.java:59)
ghidra-aarch64-darwin>         ... 11 more
ghidra-aarch64-darwin> 
ghidra-aarch64-darwin> 
ghidra-aarch64-darwin> * Get more help at https://help.gradle.org
ghidra-aarch64-darwin> 
ghidra-aarch64-darwin> 
ghidra-aarch64-darwin> * Try:
ghidra-aarch64-darwin> > Run with --stacktrace option to get the stack trace.
ghidra-aarch64-darwin> > Run with --debug option to get more log output.
ghidra-aarch64-darwin> > Run with --scan to get full insights.
ghidra-aarch64-darwin> 
ghidra-aarch64-darwin> * Get more help at https://help.gradle.org
```

--- a/pkgs/tools/security/ghidra/build.nix
+++ b/pkgs/tools/security/ghidra/build.nix
@@ -178,6 +178,7 @@ in stdenv.mkDerivation rec {
     ];
     license = licenses.asl20;
     maintainers = with maintainers; [ roblabla ];
+    broken = stdenv.isDarwin;
   };
 
 }

